styles/style.css

```diff
@@ -6,3 +6,171 @@ Green: #00B172
 White: #FFF
 
 */
+
+@import url('https://fonts.googleapis.com/css2?family=Varela+Round&display=swap');
+
+
+body{
+    
+    font-family: 'Varela Round', sans-serif;
+}
+
+
+
+h3{
+    color: #00B172;
+}
+
+.navigation-bar {
+    width:100%;
+    height: 80px;
+    position:fixed;
+    background-color: white;
+    top:0;
+}
+
+
+.logo {
+    float:left;
+    width:100px;
+    height:auto;
+    margin-left:30px;
+    margin-top: 20px;
+    vertical-align: top;
+}
+
+.nav-list {
+    float: right;
+    margin: 20px 30px;
+    list-style-type: none;
+    display: flex;  
+}
+
+li {
+    margin: 0 10px;
+}
+
+.section-1 {
+    height:700px;
+    width:100%;
+    background-image: url(../images/landing.jpg);
+    background-size:cover;
+    background-position:center;
+}
+
+.section-1 h1 {
+ margin-left: auto;
+ margin-right: auto;
+ text-align: center;
+ align-items: center;
+ font-size: 4em;
+ color: #FFF;
+ padding-top: 250px;
+ font-weight: bold;
+
+
+}
+
+.section-1 h2 {
+    margin-left: auto;
+    margin-right: auto;
+    text-align: center;
+    align-items: center;
+    font-size: 2em;
+    color: #FFF;
+    padding-top: auto;
+    width:60%;
+    font-weight:lighter;
+
+}
+.section-2{
+    margin: 40px;
+    text-align: center;
+    
+}
+.whats h2 {
+    border-bottom: 2.5px solid #00B172;
+    text-align:center;
+    display:inline;
+    font-size:2.5em;
+    font-weight:bold;
+    padding-bottom: 7px;
+    
+    
+    
+}
+.section-2{
+    margin-top: 50px;
+}
+.section-2 h2 {
+    border-bottom: 2.5px solid #00B172;
+    text-align:center;
+    margin:0 450px;
+}
+
+#icons{
+    width:75px;
+    height:75px;
+}
+
+.container{
```

Comment:
The 3 boxes here are not really centered. That's a way of achieveing this: 
`.container {`
   ` padding-top: 80px;`
   ` display: flex;`
    `justify-content: space-evenly;`
}

`.box {`
    `width: 33%;`
   ` text-align: center;`
`}`